addons/l10n_in/models/res_config_settings.py

```diff
@@ -19,11 +20,30 @@ class ResConfigSettings(models.TransientModel):
     module_l10n_in_edi_ewaybill = fields.Boolean('Indian Electronic Waybill')
     module_l10n_in_gstin_status = fields.Boolean('Check GST Number Status')
     module_l10n_in_withholding = fields.Boolean('Indian TDS and TCS')
+    module_l10n_in_reports_gstr = fields.Boolean('GSTR India eFiling')
     l10n_in_hsn_code_digit = fields.Selection(related='company_id.l10n_in_hsn_code_digit', readonly=False)
     module_l10n_in_enet_batch_payment = fields.Boolean(string="Vendor Payment")
+    l10n_in_is_gst_registered = fields.Boolean('Register Under GST', config_parameter='l10n_in.l10n_in_is_gst_registered')
+    l10n_in_gstin = fields.Char(related='company_id.vat', readonly=False)
+    l10n_in_tax_payer_type = fields.Selection(
+        selection=[
+            ('normal_taxpayer', 'Normal Taxpayer'),
+            ('composition_taxpayer', 'Composition Taxpayer'),
+            ('casual_taxable_person', 'Casual Taxable Person'),
+            ('input_service_distributor', 'Input Service Distributor (ISD)'),
+            ('non_resident_taxable_person', 'Non-Resident Taxable Person'),
+            ('online_service_distributor', 'Non-Resident Online Service Distributor'),
+            ('embassy_un_body', 'Embassy / UN Body / Other Notified Persons'),
+            ('sez_developer_unit', 'Special Economic Zone (SEZ) Developer / Unit'),
+            ('tds_tcs', 'Tax Deductor at Source (TDS) / Tax Collector at Source (TCS)'),
+        ],
+        string='Payer Type',
```

Comment:
If fields like this and the TAN are not used for anything in the code, we should not have them.  If users want to add pure informational fields, they have things like Studio.